src/db/spanner/models.rs

```diff
@@ -1609,38 +1609,38 @@ impl SpannerDb {
     }
 }
 
-unsafe impl Send for SpannerDb {}
+unsafe impl Send for SpannerDb<'_> {}
```

Comment:
We can remove this unsafe Send impl and the Send requirement from the Db trait. Tested and it compiles/runs.